docs/Kube-API.md

```diff
@@ -39,6 +48,44 @@ Here how to approve an Agent:
 $ kubectl -n assisted-installer patch agents.agent-install.openshift.io 120af504-d88e-46bd-bec2-b8b261db3b01 -p '{"spec":{"approved":true}}' --type merge
```

Comment:
for consistency, can you please add `sh` syntax highlighting as you did in line 63?